README.Rmd

```diff
@@ -323,6 +325,107 @@ val_app1 |>
               se = F)
 ```
 
+### Approach B
+The previous approach assigned a count site to each road link based. So far, it
+has not been addressed the fact that counts might be reporting uni-directional
+flows along specific links. 
+
+Using the `sf_counts_selected` object, we produce a buffer of 30 metres from
+each count site.
+```{r}
+sel_counts_buf30 = st_buffer(sf_counts_selected,dist = 30)
+```
+
+All overlaps within the `sel_counts_buf30` are identified using the following
+code. 
+```{r}
+counts_overlap = st_intersects(sel_counts_buf30, sel_counts_buf30)
+```
+
+Some overlaps might reveal count sites reporting flows of different
+directions on the same edge/link. For example, the sites EDH0040 and EDH0041
+which are located on Mayfield Road.
+
+```{r}
+tmap_mode("view")
+tm_shape(sel_counts_buf30[sel_counts_buf30$siteID %in% c("EDH0040","EDH0041"),])+
+  tm_polygons(alpha = 0.5)+
+  tm_shape(sf_counts_selected)+
+  tm_dots()
+  
+```
+
+
+A more complex instance is the overlap of sensors on Melville Dr, two of the
```

Comment:
Did this improve the counter-network fit?

Reply:
There is a small increase in the R<sup>2</sup>, although the 2011 estimates are very low (~0.7% of the counts). I have a added a linear model at the end of each approach to show this in be7440f. I am planning to include a simple relative difference and GEH comparison.